.github/workflows/osv-scanner-scan.yml

```diff
@@ -18,4 +18,4 @@ jobs:
       contents: read # to fetch code (actions/checkout)
       security-events: write # for uploading SARIF files
     if: ${{ github.repository == 'metal3-io/ip-address-manager' && github.event_name == 'schedule' }}
-    uses: google/osv-scanner-action/.github/workflows/osv-scanner-reusable.yml@4a318af4a84899723625da2927c30fcfb7247126 # v1.8.4
+    uses: google/osv-scanner-action/.github/workflows/osv-scanner-reusable.yml@7c52d44abe9736f8a11bac47f6baadad7b3389f5 # v1.8.5
```

Comment:
Put the empty line back.

Reply:
there wasn't any empty line, I have messed up the EoF it seems

Reply:
fixed